apps/vx-scan/backend/src/app.ts

```diff
@@ -225,10 +220,14 @@ function buildApi(
       return await backupToUsbDrive(store, usb);
     },
 
-    async exportCastVoteRecordsToUsbDrive(input: {
-      machineId: string;
-    }): Promise<Result<void, ExportDataError>> {
-      return await exportCastVoteRecordsToUsbDrive(store, usb, input.machineId);
+    async exportCastVoteRecordsToUsbDrive(): Promise<
+      Result<void, ExportDataError>
+    > {
+      return await exportCastVoteRecordsToUsbDrive(
+        store,
+        usb,
+        getMachineConfig().machineId
```

Comment:
Nice - loving that client calls are getting simpler as we consolidate data on the backend